pyvrp/cpp/CostEvaluator_bindings.cpp

```diff
@@ -10,10 +10,20 @@ PYBIND11_MODULE(_CostEvaluator, m)
         .def(py::init<unsigned int, unsigned int>(),
              py::arg("capacity_penalty"),
              py::arg("tw_penalty"))
+        .def_property_readonly_static(
+            "get_default",                // this is a bit of a workaround for
+            [](py::object)                // classmethods, because pybind does
+            {                             // not yet support those natively.
+                return py::cpp_function(  // See issue 1693 in the pybind repo.
+                    []() { return CostEvaluator(0, 0); });
+            })
```

Comment:
What's the intended use case for this method?

Reply:
We agreed on 0 as defaults, but I thought we could not have multiple constructor overloads or default values using the bindings so I implemented this as a helper function. We could also make it a singleton instance actually (but not sure if we can implement this in the bindings... suggestions welcome!

Reply:
I think we can have one constructor that has `0` as default. In the constructor bindings the arguments are assignable:
```
py::arg("val") = 0
```
That should work!

Reply:
I miss the 'mind-is-blown' emoticon to reply. But thanks, that's much better :)